src/ToDo.Api/Readme.md

```diff
@@ -0,0 +1,419 @@
+# Bunsen Burner 🔥
+
+A [testing library](https://github.com/bmazzarol/Bunsen-Burner) which helps to easily write 
+well-structured test cases in both `Arrange, Act, Assert (AAA)`, and `Given, When, Then (BDD)`
+testing patterns.
+
+## Context :dart:
+
+When it comes to testing .NET applications, there are many options.
+
+I think, as .NET developers, we are privileged to have such a plateau of options to choose.
+But, this can be a problem as well, since now we have too many options to choose, ranging 
+from packages, conventions, and guides.
```

Comment:
Are we saying the various testing libraries  make it harder to structure tests?

Might be a better angle to emphasize consistent and DRY.